internal/pkg/cli/task_run.go

```diff
@@ -311,7 +311,7 @@ func (o *runTaskOpts) validateFlagsWithSubnets() error {
 	}
 
 	if o.useDefaultSubnets {
-		fmt.Errorf("cannot specify both `--subnets` and `--default`")
+		return fmt.Errorf("cannot specify both `--subnets` and `--default`")
```

Comment:
wut...

Reply:
We should add a test for this, then!

Reply:
I looked into it and there is already a test for it, but the error is caught in a separate codepath above while validating the `--default` flag, i.e. this statement cannot be reached :/

I wanted to remove it but it required a refactor on how we validate flags together and wasn't related to the PR so I stopped here.

I'll address the issue when adding the golang-ci-github-action.